documentai/api/DocumentAI.Samples.Tests/DocumentAIFixture.cs

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.DocumentAI.V1;
+using Google.Protobuf.WellKnownTypes;
+using System;
+using System.Threading;
+using Xunit;
+
+[CollectionDefinition(nameof(DocumentAIFixture))]
+public class DocumentAIFixture : IDisposable, ICollectionFixture<DocumentAIFixture>
```

Comment:
If you are not currently disposing of anything, we don't need this.
```suggestion
public class DocumentAIFixture : ICollectionFixture<DocumentAIFixture>
```